.travis.yml

```diff
@@ -4,52 +4,45 @@ dist: trusty
 
 matrix:
   include:
+    - name: Lint
+      os: osx
+      env: TASK=LINT
+    - name: Code coverage
+      os: osx
+      osx_image: xcode10.1
+      env: MONGODB_VERSION=3.6.5 TASK=COV
     - os: osx
       osx_image: xcode9.4 
-      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.0
+      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.0 TASK=TEST
     - os: osx
       osx_image: xcode9.4
-      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.1
+      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.1 TASK=TEST
     - os: osx
       osx_image: xcode10.1
-      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.1
+      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.1 TASK=TEST
     - os: osx
       osx_image: xcode10.1
-      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.2
+      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.2 TASK=TEST
     - os: linux
-      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.0
+      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.0 TASK=TEST
     - os: linux
-      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.1
+      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.1 TASK=TEST
     - os: linux
-      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.2
+      env: MONGODB_VERSION=3.6.5 SWIFT_VERSION=4.2 TASK=TEST
 
 install:
-  - MONGODB_BASE="mongodb-linux-x86_64"
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then MONGODB_BASE="mongodb-osx-ssl-x86_64"; fi
-
-  # install mongodb
-  - wget http://fastdl.mongodb.org/${TRAVIS_OS_NAME}/${MONGODB_BASE}-${MONGODB_VERSION}.tgz
-  - mkdir mongodb-${MONGODB_VERSION}
-  - tar xzvf ${MONGODB_BASE}-${MONGODB_VERSION}.tgz -C mongodb-${MONGODB_VERSION} --strip-components 1
-  - ${PWD}/mongodb-${MONGODB_VERSION}/bin/mongod --version
-
-  # install swiftenv
-  - eval "$(curl -sL https://swiftenv.fuller.li/install.sh)"
-  
-  # clone and build libmongoc
-  - git clone -b r1.13 https://github.com/mongodb/mongo-c-driver /tmp/libmongoc
-  - pushd /tmp/libmongoc
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr/local; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr; fi
-  - sudo make -j8 install
-  - popd
+  - if [[ "$TASK" == "TEST" ]]; then ./install.sh libmongoc; eval "$(curl -sL https://swiftenv.fuller.li/install.sh)"; ./install.sh mongodb; fi
```

Comment:
might want to choose another name for our installer so we don't conflate it with the swiftenv installer. (or download the swiftenv installer as something other than install.sh)

Reply:
renamed `travis_install.sh`.